src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -153,6 +153,7 @@ function invokeJSFromDotNet(callInfo: Pointer, arg0: any, arg1: any, arg2: any):
         return result;
       case DotNet.JSCallResultType.JSObjectReference:
         return DotNet.createJSObjectReference(result).__jsObjectId;
+      case DotNet.JSCallResultType.JSDataReference:
```

Comment:
Is this deliberately blank? Seems like something should be here.

Reply:
Deliberately blank as we don't support streaming in WASM. The `default` case below should be sufficient, I just preferred being more explicit here. With regard to whether we should support streaming in WASM, see [here](https://github.com/dotnet/aspnetcore/pull/33491#discussion_r654429513).